### PR TITLE
Routes: Create /home /status /experiment on the frontend

### PR DESCRIPTION
**Pull request by caleybrock**
## Screenshot
react-router

## Implementation 
At first, my plan was to entirely replace create-react-app frontend with https://neutrinojs.org/, but this ended up working. It's possible we still might need to do this, but right now I can't for-see any issues.
I've used react-router to get this working: https://facebook.github.io/create-react-app/docs/adding-a-router

## Testing
I've also created a few new components with blank tests.
screen shot 2019-01-21 at 3 35 37 pm
@MichaelCherry I was unable to run the test scripts on my macbook - I think it's another issue with line endings.

### CODE REVIEW

**Comment by MichaelCherry** [comment]
This worked on my machine, but took about 3 mins to finish building frontend the first time. going to see if it is faster the second time. However, once it finished, everything seemed to work as expected.

**Comment by MichaelCherry** [comment]
This needed "docker system prune" before it built frontend on Francis's machine

**Comment by fnormand01** [comment]
As Michael mentioned in his previous comment, I had to do a "docker system prune" and then wait like 3 to 5 minutes before the front-end finishes his startup.

Before that, I was getting the following error:
image

**Comment by caleybrock** [comment]
I believe that's because I made changes to `yarn` so your system would need to do a full install. Not unexpected, but glad it's working now.

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,42 +1,30 @@
 import React, { Component } from 'react';
 import logo from './logo.png';
```
Comment (fnormand01) [inline frontend/src/App.js]: Here you have *eslint* and *ts* errors:
image

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,42 +1,30 @@
...
 import './App.css';
-
-const headers = new Headers({
-  Accept: 'application/json',
-  'Content-Type': 'application/json',
-});
+import Home from './Home.jsx';
+import Experiment from './Experiment.jsx';
+import Status from './Status.jsx';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
```
Comment (fnormand01) [inline frontend/src/App.js]: Here you have an *eslint* error:
image

Can we fix that too?

File: frontend/src/App.js [hunk 3]
```diff
@@ -1,42 +1,30 @@
...
-      headers,
-      cache: 'default',
-    });
-    const testJson = await test.json();
-    if (testJson && testJson.status) {
-      this.setState({ test: testJson.status });
-    }
-  }
-
   render() {
```
Comment (fnormand01) [inline frontend/src/App.js]: Here you have an *eslint* error:
image

Can we also fix that one?

**Comment by fnormand01** [comment]
Once all the *eslint* and *ts* errors that I commented in **App.js** will be fixed, I'm good with these changes.

**Comment by fnormand01** [comment]
Tested it again after you updates. Everything is working fine, as expected.

File: frontend/src/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (MichaelCherry) [reply frontend/src/App.js]: this is fixed with last commit

**Pull request merged**